stylesheets/_component.breadcrumb.scss

```diff
@@ -1,6 +1,8 @@
 .breadcrumb {
+    clear: both;
     color: color(gray, dark); // must be AA 4.5:1
-    margin-bottom: 11px;
+    margin: 0 0 ($line-height-base / 2);
+    width: 100%;
```

Comment:
Needs checking to see if still needed following flash changes (which included other `header.toolbar` spacing tweaks.